pkgs/applications/science/physics/crystfel/default.nix

```
@@ -169,13 +170,29 @@ let
       "-DENABLE_BZIP2_PLUGIN=yes"
     ];
   };
+
+  millepede-ii = stdenv.mkDerivation rec {
+    pname = "millepede-ii";
+    version = "V04-13-06";
+    src = fetchurl {
+      url = "https://gitlab.desy.de/claus.kleinwort/millepede-ii/-/archive/${version}/millepede-ii-${version}.tar.gz";
+      hash = "sha256-aFoo8AGBsUEN2u3AmnSpTqJ6JeNV6j9vkAFTZ34I+sI=";
+    };
+
+    nativeBuildInputs = [ gfortran ];
+    buildInputs = [ zlib ];
+
+    preBuild = ''
+      makeFlagsArray+=("PREFIX=$out")
+    '';
```

Why not use makeFlags directly?

```suggestion
    makeFlags = [
      "PREFIX=$(out)"
    ];
```

---

Not sure, but seems to work the same, fixed.